### PR TITLE
UIDEXP-213: Replace translation keys with translations on Error logs page

### PR DESCRIPTION
**Pull request by yelyzavetakhokhlova**
## Purpose
In scope of [UIDEXP-213](https://issues.folio.org/browse/UIDEXP-213).
Dynamically replace translation keys that are provided by backend with localized messages with proper translations.

## Coverage
image

### CODE REVIEW

File: src/components/ErrorLogsView/ErrorLogsView.js [hunk 1]
```diff
@@ -41,10 +57,26 @@ export const ErrorLogsViewComponent = ({ resources: { log } }) => {
   );
 };
 
+ErrorLogsViewComponent.propTypes = {
```
Comment (ViktorSoroka07) [inline src/components/ErrorLogsView/ErrorLogsView.js:60]: 👍

**Comment by ViktorSoroka07** [comment]
Please attach coverage information.

**Review by ViktorSoroka07 — APPROVED** [review]

**Comment by ViktorSoroka07** [comment]
Looks great especially several last commits 🙂.

**Comment by yelyzavetakhokhlova** [comment]
> Looks great especially the last commit🙂.

Glad it's green now! 😉

**Pull request merged**